src/InputWithLabel.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const InputWithLabel = (props) => {
+    const inputRef = React.useRef(0);
+    React.useEffect(() => {
+        inputRef.current.focus()
+    });
+
+
+    return (
+        <>
+            <label htmlFor='todoTitle'>{props.children}</label>
+            <input autoFocus id="todoTitle" name="title" value={props.todoTitle} onChange={props.handleTitleChange} ref={inputRef}></input>
+        </>
```

Comment:
Please consider passing all the attributes for the input element as `props`. This approach allows you to enhance the reusability of your `InputWithLabel` component.